### PR TITLE
store/tikv, config: config for grpc connection.

### PR DESCRIPTION
**Pull request by disksing**

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -147,6 +148,13 @@ type ProxyProtocol struct {
...
 }
 
+// TiKVClient is the config for tikv client.
+type TiKVClient struct {
+	// GrpcConnectionCount is the max gRPC connections that will be established
+	// with each tikv-server.
+	GrpcConnectionCount int `toml:"grpc-connection-count" json:"grpc-connection-count"`
+}
+
 var defaultConf = Config{
```
Comment (coocood) [inline config/config.go]: Why not add it to defaultConf?

File: config/config.toml.example [hunk 2]
```diff
@@ -191,3 +191,7 @@ log-spans = false
 
 #  LocalAgentHostPort instructs reporter to send spans to jaeger-agent at this address
 local-agent-host-port = ""
+
+[tikv-client]
+# Max gRPC connections that will be established with each tikv-server.
+# grpc-connection-count = 16
```
Comment (coocood) [inline config/config.toml.example]: uncomment it if it is default value.

File: config/config.go (hunk 1 above)
Comment (disksing) [reply config/config.go]: defaultConf does not use tikv store.

Comment (coocood) [reply config/config.go]: You can see the defalt config has a lot of `Enabled: false`, but still has other default values set.

**Comment by disksing** [comment]
PTAL @coocood @zimulala

**Comment by coocood** [comment]
LGTM

**Comment by zimulala** [comment]
LGTM

**Pull request merged**